server/src/main/java/org/elasticsearch/action/support/replication/TransportReplicationAction.java

```diff
@@ -96,6 +98,21 @@
             Response extends ReplicationResponse
         > extends TransportAction<Request, Response> {
 
+    /**
+     * The timeout for retrying replication requests.
+     */
+    public static final Setting<TimeValue> REPLICATION_RETRY_TIMEOUT =
+        Setting.timeSetting("indices.replication.retry_timeout", TimeValue.timeValueSeconds(0), Setting.Property.Dynamic,
```

Comment:
I have adjusted this to `0` to prevent disable retries until back pressure is implemented.